scripts/delayed.js

```diff
@@ -58,7 +61,36 @@ async function loadRelatedContent() {
   rcHeader.insertAdjacentElement('afterend', rcTeasers);
 }
 
+async function loadYMAL() {
+  const placeholders = await fetchPlaceholders(`/${getLocale()}`);
+  const { ymal } = placeholders;
+  const mltSection = document.createElement('div');
+  mltSection.className = 'section box-shadow post-teaser-list-container';
+  const mltWrapper = document.createElement('div');
+  mltWrapper.className = 'default-content-wrapper';
+  mltSection.append(mltWrapper);
+  const ptlBlock = document.createElement('div');
+  ptlBlock.className = 'post-teaser-list cards';
+  mltWrapper.append(ptlBlock);
+  ptlBlock.innerHTML = `<div>
```

Comment:
Will text remain same for other locales ?

Reply:
good catch. that was some legacy code I was using for testing.